src/app/app.component.html

```diff
@@ -1,7 +1,23 @@
 <div class="flex-row margin-row">
-	<form onsubmit="return false">
-		<app-date-range-picker [formControl]="dateRangeFormControl"></app-date-range-picker>
-	</form>
+	<div class="flex-1">
+		<form class="date-wrapper" [formGroup]="dateRangeFormGroup" onsubmit="return false">
+			<!-- date picker -->
+			<app-date-range-picker formControlName="dateRangeFormControl"></app-date-range-picker>
+
+			<span>or</span>
+
+			<!-- date dropdown -->
+			<div class="date-wrapper-dropdown">
```

Comment:
Let's wrap with a component that implements control value accessor (and validator if needed). Let's skip using a directive to override control value accessor (and validator if needed).